### PR TITLE
Change entity Address to Location. #50.

### PR DESCRIPTION
**Pull request by RasmusBroborg**

### CODE REVIEW

File: src/GT.Core/Services/Impl/GTLocationService.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿using GT.Core.DTO;
+using GT.Core.Services.Interfaces;
+using GT.Data.Data.GTAppDb.Entities;
+using GT.Data.Repositories;
+
+namespace GT.Core.Services.Impl
+{
+	public class GTLocationService : GTGenericService<City, LocationDTO>, IGTLocationService
```
Comment (JohanSaisa) [inline src/GT.Core/Services/Impl/GTLocationService.cs:8]: City class should be Location

File: src/GT.Core/Services/Impl/GTLocationService.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+	{
+		public GTLocationService(IGTGenericRepository<City> repository) : base(repository)
```
Comment (JohanSaisa) [inline src/GT.Core/Services/Impl/GTLocationService.cs:10]: City class should be Location

File: src/GT.Core/Services/Impl/GTLocationService.cs [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+		{
+		}
+
+		protected override City DTOToEntity(LocationDTO dataTransferObject)
+		{
+			throw new NotImplementedException();
+		}
+
+		protected override LocationDTO EntityToDTO(City entity)
```
Comment (JohanSaisa) [inline src/GT.Core/Services/Impl/GTLocationService.cs:19]: City class should be Location

**Review by JohanSaisa — CHANGES_REQUESTED** [review]
Change City to Location in GTLocationService.cs

**Review by JohanSaisa — APPROVED** [review]
Approve.

**Review by JohanSaisa — APPROVED** [review]
Approve delta changes.

**Review by OscarThorburn — APPROVED** [review]

**Comment by RasmusBroborg** [comment]
Merge errors appear to be connected to previous PR.

**Pull request merged**